data/tilesets/buildings/occupationgov.license

```diff
@@ -0,0 +1,6 @@
+SPDX-License-Identifier: GPL-3.0-or-later
+SPDX-FileCopyrightText: 2010 Tim F. Smith <yoohootim@hotmail.com>
+SPDX-FileCopyrightText: 2010 Frederic Rodrigo <f.rodrigo@tuxfamily.org>
+SPDX-FileCopyrightText: 2010 Dusan Hukuv <dusan@mirc.cz>
+SPDX-FileCopyrightText: 2022 James Robertson <jwrober@gmail.com>
+SPDX-FileCopyrightText: 2022 Louis Moureaux <m_louis30@yahoo.com>
```

Comment:
Thanks but copyright doesn't protect ideas. Maybe could be protected if I had registered a trademark, but even then I doubt it :joy: